contracts/LpSugar.vy

```diff
@@ -689,28 +718,29 @@ def _positions(
             break
 
         # Fetch locked V2/Basic positions
-        v2_locker_factory: ILockerFactory = ILockerFactory(staticcall self.v2_launcher.lockerFactory())
-        lockers: DynArray[address, MAX_POSITIONS] = staticcall v2_locker_factory.lockersPerPoolPerUser(pool_addr, _account)
+        if self.v2_launcher.address != empty(address):
```

Comment:
I think we can DRY a bit all of this and move it into a separate function in the shared/helper contract